### ISSUE TITLE
Digital Accessibility Through Open Source Collaboration

### ISSUE THREAD

**Issue by mmmavis** [mozilla/mozfest-program#340]
**[ Google Spreadsheet Row Number ]** 304
**[ Facilitator ]** Mike Gifford
### Description

Leverage our experience with Drupal to think about how to make open source software more accessible for everyone. Web accessibility is a big issue, and will only get bigger with an ageing population. By working with people with disabilities we work to a web that is perceivable, operable and understandable to more and more people. 
### Agenda

More like a fireside chat or facilitated discussion. I want to engage with participants to learn from them about their challenges making their products accessible. I want to share what we've learned in making Drupal the most accessible open source CMS out there. 
### Participants

5 - roundtable discussion of knowledge & interests in web accessibility. Discussion about where their concerns are and how to take the next steps.

15 - round of introductions, try to determine if we can break the group into smaller groups for more in-depth discussions. highlight folks with knowledge in accessibility issues and who might be able to take a leadership role in in-depth discussions.

50 - show of hands for interests in open source projects, presentation to raise awareness about WCAG, ATAG & what we've done with Drupal 8. Discussion about how this might apply to other open projects. 
### Outcome

Hopefully blogs, but hopefully also further engagement with people with disabilities. I would be advocating for people to use basic validation tools to review projects that they use for accessibility concerns. For some sizes of groups we should be able to submit bugs to various open source projects.

**Comment by Saallen** [comment]
```
We appreciated this submission but unfortunately this session does not fit within the narrative of our Space for 2015- Hope you will submit another session for 2016
```
